pkgs/by-name/pr/pre-commit/package.nix

```diff
@@ -45,26 +48,35 @@ buildPythonApplication rec {
     virtualenv
   ];
 
-  nativeCheckInputs = [
-    cargo
-    coursier
-    dotnet-sdk
-    gitMinimal
-    glibcLocales
-    go
-    libiconv # For rust tests on Darwin
-    nodejs
-    perl
-    pytest-env
-    pytest-forked
-    pytest-xdist
-    pytestCheckHook
-    re-assert
-    cabal-install
-  ];
-
-  # i686-linux: dotnet-sdk not available
-  doCheck = stdenv.buildPlatform.system != "i686-linux";
+  nativeCheckInputs =
+    [
+      cargo
+      gitMinimal
+      glibcLocales
+      go
+      libiconv # For rust tests on Darwin
+      perl
+      pytest-env
+      pytest-forked
+      pytest-xdist
+      pytestCheckHook
+      re-assert
+      cabal-install
+    ]
+    ++ lib.optionals (!i686Linux) [
```

Comment:
This should really have been using availableOn and those packages should have been marked broken on i686.